frontend/app/src/main/res/layout/activity_session.xml

```diff
@@ -6,4 +6,50 @@
     android:layout_height="match_parent"
     tools:context=".SessionActivity">
 
+    <TextView
```

Comment:
Would prefer to use chains for the buttons (or a table), as that will allow us to reposition them more easily. This is ok for now